### PR TITLE
feat: change preview to beta tag in all services

### PR DESCRIPTION
**Pull request by pivotal-marcela-campo**
[#182285965](https://www.pivotaltracker.com/story/show/182285965)

### Checklist:

* [x] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +52,8 @@ BROKER_DOCKER_OPTS=--rm -v $(PAK_CACHE):$(PAK_CACHE) -v $(PWD):/brokerpak -w /br
 		-e "DB_TYPE=sqlite3" \
 		-e "DB_PATH=/tmp/csb-db" \
 		-e PAK_BUILD_CACHE_PATH=$(PAK_CACHE) \
-		-e GSB_PROVISION_DEFAULTS
+		-e GSB_PROVISION_DEFAULTS \
+		-e GSB_COMPATIBILITY_ENABLE_BETA_SERVICES=$(GSB_COMPATIBILITY_ENABLE_BETA_SERVICES)
```
Comment (zucchinidev) [inline Makefile:56]: thoughts (no-blocking): If we name an environment variable without specifying a value, then the current value of the named variable is propagated into the container's environment:

`
$ export today=Wednesday
$ docker run -e "deep=purple" -e today --rm alpine env
`
image

File: aws-postgresql.yml [hunk 2]
```diff
@@ -15,45 +15,46 @@
...
     cores: 8
     postgres_version: 11
     storage_gb: 20
     subsume: false
 - name: subsume
   id: 886cb524-1d4f-11eb-b35f-8ba96a9085d7
-  description: 'Subsume control of an existing Postgre Database'
-  display_name: "Subsume"
+  description: 'Beta - Subsume
+  control of an existing Postgre Database'
```
Comment (zucchinidev) [inline aws-postgresql.yml:56]: thoughts: probably there is an extra "\n" character

File: integration-tests/mysql_test.go [hunk 3]
```diff
@@ -17,6 +18,34 @@ var _ = Describe("MySQL", Label("MySQL"), func() {
...
+				MatchFields(IgnoreExtras, Fields{"Name": Equal("medium")}),
+				MatchFields(IgnoreExtras, Fields{"Name": Equal("large")}),
+			),
+		)
+
+		Expect(service.Plans).To(
+			HaveEach(
+				MatchFields(IgnoreExtras, Fields{
+					"Description": HavePrefix("Beta -"),
+					"Metadata": PointTo(MatchFields(IgnoreExtras, Fields{"DisplayName": HaveSuffix("(Beta)")})),
```
Comment (zucchinidev) [inline integration-tests/mysql_test.go:43]: praise: fantastic combination of matchers

**Review by zucchinidev — APPROVED** [review]
I've created some comments but none of them is blocking

File: Makefile (hunk 1 above)
Comment (pivotal-marcela-campo) [reply Makefile]: done

File: aws-postgresql.yml (hunk 2 above)
Comment (pivotal-marcela-campo) [reply aws-postgresql.yml]: done, thanks!

**Pull request merged**